cronjob/values.yaml

```diff
@@ -0,0 +1,23 @@
+image:
+  name: ''
+  repository: ''
+  tag: ''
+  pullPolicy: 'ifNotPresent'
+
+## Configure resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+resources: {}
+#  requests:
+#    cpu: 50m
+#    memory: 256Mi
+#  limits:
+#    cpu: 150m
+#    memory: 512Mi
+
+schedule: '*/1 * * * *'
```

Comment:
nit pick: all params should have a comment

Reply:
https://github.com/commercetools/k8s-charts/pull/1/commits/52c5882e3a356809cbcd87700fb8d7d4f8ebca2c